cms/models/Item.js

```diff
@@ -41,6 +41,13 @@ Item.add({
 		required: true,
 		initial: true,
 	},
+	evidenceType: {
+		type: Types.Relationship, 
+		ref: "EvidenceType",
+		initial: true,
```

Comment:
Should this initial bit be set to true? I don't think evidence types is one of the ones we were going to have on the initial page

Reply:
I haven't seen that requirement, but to me it's absolutely necessary because the content is grouped by evidence type in the email so it doesn't make sense to be able to make content items without an evidence type

Reply:
We previously cut down the number of items on that screen to just fields that were required to distinguish one record from another - there are other fields that are required for the email that also only exist on the main page. The first page acts funny when it gets too long which is why we tried to keep as many items as possible off it (ideally it wouldn't exist at all and you'd go straight into the full record).

Reply:
Makes sense, but surely better to keep evidence type but remove short summary?